doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
I think we need to pass the PV name in so that backup will do the claim, mount process to retrieve the data. Otherwise, we don't know the location of the backup data.